utility/src/main/java/org/oppia/util/logging/EventBundleCreator.kt

```diff
@@ -55,12 +54,12 @@ class EventBundleCreator {
   /** Returns a bundle from event having question context. */
   private fun createQuestionContextBundle(eventLog: EventLog): Bundle {
     val bundle = Bundle()
+    val skillIdList = eventLog.context.questionContext.skillIdList
     bundle.putLong(TIMESTAMP_KEY, eventLog.timestamp)
     bundle.putString(QUESTION_ID_KEY, eventLog.context.questionContext.questionId)
-    bundle.putStringArray(
-      SKILL_LIST_ID_KEY,
-      eventLog.context.questionContext.skillIdList.toTypedArray()
-    )
+    for (skillId in skillIdList) {
+      bundle.putString(SKILL_ID_KEY + skillIdList.indexOf(skillId), skillId)
```

Comment:
I thought we discussed that we would convert the array to a string and put it once, instead of keeping separate keys..
i.e. 
```
bundle.putString(SKILL_ID_KEY, skillIdList.joinToString())
```

Reply:
Done.